easybuild/easyconfigs/f/FFmpeg/FFmpeg-3.3-intel-2017a.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'ConfigureMake'
+
+name = 'FFmpeg'
+version = '3.3'
```

Comment:
There's already an easyconfig for 3.3.1, why roll back to an older version?

Right now, there's a 3.3.2 available already too...